libs/ui/atoms/src/lib/Icon/Icon.tsx

```diff
@@ -80,126 +80,139 @@ enum IconType {
 }
 
 interface IIconProps {
+  size?: number | undefined;
+  color?: string | undefined;
+  backgroundColor?: string | undefined;
```

Comment:
We are using NativeBase components, could we have the same props naming? https://docs.nativebase.io/utility-props

Reply:
No, we cant because we are using this library https://github.com/oblador/react-native-vector-icons. 
We want to have as coherent a whole as possible.